transform-plugins/pom.xml

```diff
@@ -76,6 +76,11 @@
       <groupId>co.cask.cdap</groupId>
       <artifactId>cdap-unit-test</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <version>20090211</version>
```

Comment:
Might want to move the version to top level to make it consistent and avoid other sub-projects to use different version.